src/Util/ThreadFactory.php

```diff
@@ -6,21 +6,27 @@
 
 class ThreadFactory
 {
-    /** @var array|Thread[] */
+    /**
+     * @var array|Thread[]
+     */
     private $threadList;
 
     /**
      * ThreadFactory constructor.
+     *
      * @param array $threads array of threads as retrieved from API
      * @param $board
      * @param bool $useSSL True for Https, false for Http
+     * @return void
      */
     public function __construct($threads, $board, $useSSL = true)
     {
         $this->threadList = $this->flattenThreads($threads, $board, $useSSL);
     }
 
     /**
+     * Get all the threads!
```

Comment:
This needs to be all active threads
(Other API calls support grabbing archived threads)